File: src/main/java/site/gachontable/gachontablebe/domain/admin/usecase/GetSeatings.java
package site.gachontable.gachontablebe.domain.admin.usecase;

import site.gachontable.gachontablebe.domain.admin.presentation.dto.response.SeatingsResponse;
import site.gachontable.gachontablebe.domain.auth.domain.AuthDetails;

public interface GetSeatings {

    SeatingsResponse execute(AuthDetails authDetails);
}

File: src/main/kotlin/site/gachontable/domain/admin/port/in/GetSeatings.kt
package site.gachontable.domain.admin.port.`in`

import site.gachontable.infra.security.principal.AuthDetails
import site.gachontable.presentation.admin.dto.response.SeatingsResponse

interface GetSeatings {
    fun execute(authDetails: AuthDetails): SeatingsResponse
}
